wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/commands/DriveDistance.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.examples.romireference.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.examples.romireference.Drivetrain;
```

Comment:
```suggestion
import edu.wpi.first.wpilibj.examples.romireference.subsystems.Drivetrain;
```

Reply:
Yeah missed this. Not sure how it built on my laptop but tested and commited. Please review.